src/axom/primal/operators/detail/intersect_ray_impl.hpp

```diff
@@ -41,23 +42,48 @@ inline bool intersect_ray(const primal::Ray<T, 2>& R,
 {
   AXOM_STATIC_ASSERT(std::is_floating_point<T>::value);
 
-  // STEP 0: Find the parameterized direction of the segment
+  // STEP 0: Find the parameterized direction of the segment.
+  //         efine lower/upper threshold
```

Comment:
Delete this line?

Reply:
Fixed now, along with some of the other debugging comments. Thank you!